Assets/Shopify/UIToolkit/Components/RemoteImageLoader.cs

```diff
@@ -46,20 +46,26 @@ public class RemoteImageLoader : MonoBehaviour {
             LoadImageURL(imageURL, (texture, error) => {
                 if (error != null) {
                     Debug.LogWarning("Failed to download image at " + imageURL + " Reason: " + error);
-                    failure(error);
+                    if (failure != null) {
+                        failure(error);
+                    }
                     return;
                 }
 
                 if (texture == null) {
                     Debug.LogWarning("Failed to generate texture from image located at " + imageURL);
-                    failure("Texture not found");
+                    if (failure != null) {
+                        failure("Texture not found");
+                    }
                     return;
                 }
 
                 _image.sprite = SpriteFromTexture(texture);
                 _image.preserveAspect = true;
 
-                success();
+                if (success != null) {
```

Comment:
First time actually using the image loader and was running into NPE whenever calling success here with null callbacks.